app/assets/javascripts/discourse/scripts/async-stylesheets.js

```diff
@@ -0,0 +1,22 @@
+// This script is inlined in `_discourse_stylesheet.html.erb
+const links = document.getElementsByClassName("async-css-loading");
+
+const processEvent = function (element) {
+  delete element.dataset["media"];
+  element.dataset["processed"] = true;
+};
+
+[...links].forEach(function (element) {
+  element.addEventListener(
+    "error",
+    function () {
+      processEvent(element);
```

Comment:
Shall we call `element.removeEventListener` here and on "load" 5 lines below?

Reply:
This is a good point. While I passed the `once: true` option but that will only remove the event listener when it is invoked. If it is not invoked, it is never called.